testdata/dsp-ingest_data/dsp-ingest.json

```diff
@@ -0,0 +1,92 @@
+{
+  "$schema": "https://raw.githubusercontent.com/dasch-swiss/dsp-tools/main/src/dsp_tools/resources/schema/project.json",
+  "project": {
+    "shortcode": "00A2",
+    "shortname": "dog",
+    "longname": "Ontology for Dogs",
+    "descriptions": {
+      "en": "Ontology for Dongs"
+    },
+    "keywords": ["Dog"],
+    "lists": [],
+    "ontologies": [
+      {
+        "name": "dog",
+        "label": "Ontology for Dogs",
+        "properties": [
+          {
+            "name": "hasImage",
+            "super": ["hasLinkTo"],
+            "object": ":Image2D",
+            "labels": {"en": "Image"},
+            "gui_element": "Searchbox"
+          },
+          {
+            "name": "hasName",
+            "super": ["hasValue"],
+            "object": "TextValue",
+            "labels": {"en": "Name"},
+            "gui_element": "Richtext"
+          },
+          {
+            "name": "hasTitle",
+            "super": ["hasValue"],
+            "object": "TextValue",
+            "labels": {"en": "Title"},
+            "gui_element": "SimpleText"
+          },
+          {
+            "name": "hasAttribution",
+            "super": ["hasValue"],
+            "object": "TextValue",
+            "labels": {"en": "Attribution"},
+            "gui_element": "SimpleText"
+          },
+          {
+            "name": "hasSourceLink",
+            "super": ["hasValue"],
+            "object": "UriValue",
+            "labels": {"en": "External link"},
+            "gui_element": "SimpleText"
+          }
```

Comment:
to keep the test to the point, I don't think you need any property except `hasImage`, right?

Reply:
The reason for the test data as it is, is that I want to have an image in a parent folder, one in a subfolder and one in each that has the same name. This way I think we covered the everything. Regarding the other properties, although the images are in the public domain wikimedia asks to add a attribution. Since this will be publicly available, I wanted to do it correctly. But I can delete the hasName.

Reply:
I understand about the simplicity. I would however like to have distinct pictures that I can differentiate from each other. I took care to download the lowest quality. It was easier than taking my own pictures and reducing the quality...